### ISSUE TITLE
M1 and Pre-2012 Mac support for A7 devices using iPwner32

### ISSUE THREAD

**Issue by Grandboy46** [MatthewPierson/Ramiel#3]
Hey guys, I have an M1 Macbook Pro and 2010 iMac but both are not supported by ramiel. I know a Pwndfu tool called iPwner32 which supports a7 devices. Please add support by iPwner32 to these macs.

**Comment by MatthewPierson** [comment]
If iPwnder32 works on your machines then you can just use Ramiel fine. Theres not anything set to make it not run on older macs (or M1 macs technically), I just don't officially support them so if it works it works, if it doesn't it doesn't

**Comment by dora2-iOS** [comment]
@Grandboy46 Yes, iPwnder32 may not work on older Mac. However, the cause is unknown.
If you have the problem, please post an issue [here](https://github.com/dora2-iOS/iPwnder32/issues). If that solves the iPwnder32 problem, I think it can be solved by porting it to Ramiel.
